packages/core/android/src/test/kotlin/com/datadog/tools/unit/GenericAssert.kt

```diff
@@ -49,6 +49,12 @@ class GenericAssert(actual: Any) :
         return this
     }
 
+    fun hasFieldClassEqualTo(name: String, expected: String): GenericAssert {
```

Comment:
You should rename this function as `hasFieldWithClass`, and maybe have the param renamed to `expectedClassName` for clarity.

Reply:
Deal thanks, will change